mgmtsystem_action/__openerp__.py

```diff
@@ -20,31 +20,25 @@
 ##############################################################################
 {
     "name": "Management System - Action",
-    "version": "8.0.1.2.0",
+    "version": "9.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Management System",
-    "description": """\
-This module enables you to manage the different actions of your management
-system:
-  * immediate actions
-  * corrective actions
-  * preventive actions
-  * improvement opportunities.
-""",
-    "depends": ['mgmtsystem', 'crm_claim'],
+    "depends": ['mgmtsystem', 'crm_claim', 'mail'],
```

Comment:
Dear daniel
Are you saying u can delete it right ?

Regards
Christophe Ama OPS Manager
Sent by iPhone

> Le 22 mai 2016 à 17:28, Daniel Reis <notifications@github.com> a écrit :
> 
> In mgmtsystem_action/__openerp__.py:
> 
> >      "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
> >      "website": "http://www.savoirfairelinux.com",
> >      "license": "AGPL-3",
> >      "category": "Management System",
> > -    "description": """\
> > -This module enables you to manage the different actions of your management
> > -system:
> > -  * immediate actions
> > -  * corrective actions
> > -  * preventive actions
> > -  * improvement opportunities.
> > -""",
> > -    "depends": ['mgmtsystem', 'crm_claim'],
> > +    "depends": ['mgmtsystem', 'crm_claim', 'mail'],
> I'm not sure the dependency on crm_claim is really needed. Can you check?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>